### PR TITLE
Run rpmgrill in Travis via Docker

### PR DESCRIPTION
**Pull request by romanofski**
This adds a minimal travis configuration using docker to run rpmgrill
tests in a Fedora 24 environment.

The environment runs as privileged in order to allow tests to write temporary files into the working directory.

### CODE REVIEW

File: dev-tools/docker/ci/f24/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+FROM fedora:24
+MAINTAINER rpmdiff-maint@redhat.com
+
+### code ###
+VOLUME ["/code"]
```
Comment (sthaha) [inline dev-tools/docker/ci/f24/Dockerfile]: why is this needed?

Comment (romanofski) [reply dev-tools/docker/ci/f24/Dockerfile]: I pinched parts of this and did not realise that I don't need to specify it if I'm not sharing the containers around. Will remove.
